resources/lang/en/navigation.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 return [
-    'sidebar-label' => 'Roles & Permissions',
+    'sidebar-label' => 'Access',
```

Comment:
> And roles & permissions are both considered "Access Control"

Why "Access" though?

Reply:
It's short, it doesn't break on two lines and it's what roles & permissions are both categorized under. "Access Control Layer"